packages/commonjs/test/test.js

```diff
@@ -393,20 +393,6 @@ test('does not process the entry file when it has a leading "." (issue #63)', as
   await t.notThrowsAsync(executeBundle(bundle, t));
 });
 
-test('does not reexport named contents', async (t) => {
```

Comment:
can you comment on why this test was removed?

Reply:
It tests that the plugin throws on unknown named exports. This is exactly what the syntheticNamedExports enables.

Reply:
I see. (forgive my ignorance here, the commonjs plugin isn't all that familiar yet.) would we want to allow people to disable `syntheticNamedExports`?

Reply:
No problem! That's indeed something to consider. Personally I'd say no, commonjs is a dynamic module format by design and this aligns the behavior of rollup with other bundlers like webpack. I can't think of a valid use case for not wanting this behavior.

Reply:
Makes sense. We'd probably do a major version on this, and if anyone complained _maybe_ take a second look at the ability to turn it off.